### PR TITLE
Add user assignments dropdown for experiment table [SCI-7440]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-7440](https://biosistemika.atlassian.net/browse/SCI-7440)

### What was done
Add user assignments dropdown for experiment table

### CODE REVIEW

File: app/services/experiments/table_view_service.rb [hunk 1]
```diff
@@ -131,10 +131,10 @@ def assigned_presenter(my_module)
                             })
       end
 
-      result[:more_users_title] = user_names_with_roles(users[4..].to_a) if users.length > 3
-
+      result[:more_users_title] = users[4..].map { |u| u.full_name }.join('&#013;').html_safe if users.length > 4
```
Comment (okriuchykhin) [inline app/services/experiments/table_view_service.rb:134]: User's name becomes unsinitiezed?

Comment (aignatov-bio) [reply app/services/experiments/table_view_service.rb:134]: It's the same way as in `user_names_with_roles` method, maybe then we need also to update there?

Comment (okriuchykhin) [reply app/services/experiments/table_view_service.rb:134]: Yes, please

File: app/assets/javascripts/experiments/table.js [hunk 2]
```diff
@@ -296,9 +320,13 @@ var ExperimnetTable = {
     this.initAccessModal();
     this.initArchiveMyModules();
     this.initManageColumnsModal();
+<<<<<<< HEAD
     this.initNewTaskModal(this);
     this.initMyModuleActions();
     this.updateExperimentToolbar();
+=======
+    this.initManageUsersDropdown();
+>>>>>>> 7a5504852 (Add user assignments dropdown for experiment table [SCI-7440])
```
Comment (okriuchykhin) [inline app/assets/javascripts/experiments/table.js:329]: @aignatov-bio there is a problem here

**Review by okriuchykhin — APPROVED** [review]

**Pull request merged**